src/OSX/Avalonia.MonoMac/Helpers.cs

```diff
@@ -27,5 +28,14 @@ public static CGPoint ConvertPointY(this CGPoint pt)
             return new CGPoint(pt.X, t - pt.Y);
         }
 
+        public static Rect ConvertRectY(this Rect rect)
+        {
+            return new Rect(new Point(rect.X, rect.Position.ConvertPointY().Y), rect.Size);
```

Comment:
This conversion is not correct. In original coordinate system Y is the position of the *bottom* side of the rectangle. In our windows-like coordinate system Y is the position of the *top* side of the rectangle.